courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/2_dataset_api.ipynb

```diff
@@ -106,6 +106,17 @@
     "We will do that here."
```

Comment:
<p>bathces -&gt; batches in 2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>done</p>